librz/analysis/d/types.sdb.txt

```diff
@@ -1763,11 +1763,11 @@ func.quick_exit.args=1
 func.quick_exit.arg.0=int,status
 func.quick_exit.ret=void
 
-*aligned_alloc=func
```

Comment:
@thestr4ng3r @ret2libc not sure why is this invalid function name was in our types SDB. I converted it to the normal name. Let me know if this was intentional.

Reply:
Maybe copypasta mistake from `void *aligned_alloc(...)`, anyway `aligned_alloc` is the correct name so this is fine 🔥🐶🔥